### DISCUSSION TITLE
Schedule for cloud updates?

### DISCUSSION THREAD

**Discussion by stoberbryan** [directus/directus#14301] [Archive]
What is the schedule for cloud updates? We are still on 9.12.2 on the cloud.
Seems payers should get dibs?

**Comment by benhaynes** [comment]
@stoberbryan — Currently, Cloud gets the latest version one week after the core release. This is intended to get our vast community's eyeballs on the changes before pushing it to Cloud. We're about to push a big testing suite update as well as Nightly Builds, which will give us the ability to sync Core+Cloud so that Cloud gets updated alongside Core releases.

CC @rijkvanzanten to keep me honest on all this.
